plugin.video.mytv_bg/addon.py

```diff
@@ -15,66 +20,132 @@
 import urllib2
 import datetime
 import time
+import re
+
+__addon__           = xbmcaddon.Addon()
+__addon_name__      = __addon__.getAddonInfo('name')
+__id__              = __addon__.getAddonInfo('id')
+__lang__            = __addon__.getLocalizedString
+__settings__        = xbmcaddon.Addon(id=__addon__.getAddonInfo('id'))
+__version__         = __addon__.getAddonInfo('version')
+__profile_path__    = xbmc.translatePath( __addon__.getAddonInfo('profile') ).decode("utf-8")
+__token_filepath__  = __profile_path__ + '/token.txt'
+
+
+class Plugin(object):

```

Comment:
isn't this the default code for script.module.routing? Consider to use it as a dependency instead of repeating the code

Reply:
I did try using the default module but script.module.routing is not working properly with KODI 18 yet. If it is in the next update, I would be happy to change it.

Reply:
It does work by following the documentation update https://github.com/tamland/kodi-plugin-routing/pull/6.

The python interpreter reuse is off by default since yesterday (same behaviour as older Kodi versions), so you do not have to change anything.

Reply:
Sorry, hadn't seen that. Will rebase with the script.module.routing 0.2.1